src/javascript/JContent/ContentRoute/ContentBreadcrumb/ContentPath/CompositePathEntry.jsx

```diff
@@ -18,14 +18,15 @@ const CompositePathEntry = ({items, onItemClick}) => {
     };
 
     return (items.length > 0) &&
-        <div ref={menuContainerEl}>
```

Comment:
Shouldn't it be "data-sel-role" ?

Reply:
I used "data-role" everywhere instead, see https://github.com/Jahia/selenium/pull/189
There is no reason to use a naming that references selenium here... "data-role" is more generic.